technical_guides/chef.md

```diff
@@ -44,7 +44,7 @@
 More Information / References
 =============================
 
-  * [The Berkshelf Way ~ Jamie Winsor](http://www.getchef.com/blog/chefconf-talks/the-berkshelf-way-jamie-winsor/)
+  * The Berkshelf Way ~ Jamie Winsor
```

Comment:
If we don't have a link for this anymore, should we just remove it? I did a quick search and found [this YouTube video](https://www.youtube.com/watch?v=Dq_vGxd-jps) that's a follow-up on that talk, and this follow-up is 9 years old now. I realize that getting rid of outdated stuff is beyond the scope of this PR, but maybe this broken link specifically can go?

Reply:
I removed the link.  Citing it as a reference though is still valid I think.

Reply:
agreed